core/src/main/java/org/codice/imaging/nitf/core/tre/TreGroup.java

```diff
@@ -71,12 +72,41 @@
      Get the field value for a specific tag in integer format.
 
      @param tagName the name (tag) of the field to look up.
-     @return the field value corresponding to the tag name, as an integer.
+     @return the field value corresponding to the tag name, as a integer.
      @throws NitfFormatException when the tag is not found or the value cannot be converted to integer format.
      */
     int getIntValue(String tagName) throws NitfFormatException;
 
     /**
+     * Get the field value for a specific tag in long integer format.
+     *
+     * @param tagName the name (tag) of the field to look up.
+     * @return the field value corresponding to the tag name, as a long integer.
+     * @throws NitfFormatException when the tag is not found or the value cannot
+     * be converted to long integer format.
+     */
+    long getLongIntegerValue(String tagName) throws NitfFormatException;
```

Comment:
I wondered if it might look like API for the "decoded" value (in the sense that the short value is the abbreviation, and the long value is the full value). I agree it should at least be getLongIntValue() for consistency though.